apps/condo/k6/src/registerBillingReceiptFile.test.ts

```diff
@@ -0,0 +1,184 @@
+import { faker } from '@faker-js/faker/locale/ru'
+import dayjs from 'dayjs'
+import { check } from 'k6'
+import crypto from 'k6/crypto'
+import encoding from 'k6/encoding'
+
+import {
+    setupCondoAuth,
+    createOrganization,
+    createBillingIntegration,
+    createBillingIntegrationOrganizationContext,
+    sendAuthorizedRequest,
+    registerBillingReceiptFile,
+} from './utils'
+
+const RECEIPT_PER_ACCOUNT_NUMBER = 50
+const TOTAL_RECEIPT_NUMBER = 100
+const DURATION = '60s'
+
+export const options = {
+    tags: { testid: 'registerBillingReceiptFile', serverUrl: __ENV.BASE_URL },
+    scenarios: {
+        registerBillingReceiptFile: {
+            exec: 'registerBillingReceiptFileCase',
+            executor: 'constant-vus',
+            duration: DURATION,
+            vus: 1,
+        },
+    },
+    thresholds: {
+        http_req_failed: ['rate<0.01'],
+        http_req_duration: ['med<5000'],
```

Comment:
Why exactly 5 seconds?

Reply:
this is a high load test and proceeding of 10mb files can take few seconds for dev/local env to proceed.